### ISSUE TITLE
linux - can we add udev rules so sudo permissions aren't needed on ubuntu?

### ISSUE THREAD

**Issue by dmiddlecamp** [spark/spark-cli#184]
re: this thread
https://community.spark.io/t/usb-connection-problems-ubuntu-14-04-solved-use-sudo/10693/10

**Comment by nrobinson2000** [comment]
Yes, this would be great, to add, or at least include in the documentation, because I'm not sure if the device id **{idProduct}**, and your manufacturer id **{idVendor}** will always be the same.  I feel proud contributing to Spark.  I have used Spark Core for a year, and I am waiting for my Photons to arrive.

**Comment by emilyrose** [comment]
This would be fairly straightforward to do with what we've already done with supporting multiple vendor/device IDs in preparation for the Photon. Perhaps we should add a prompt when the CLI detects linux, that allows the user to just "press y" to have udev perms added?

**Comment by nrobinson2000** [comment]
Or have a command like **spark udev** that allows you to enter your vendor and device ID and adds a udev rule from it.
